lib/game.ex

```diff
@@ -124,4 +128,128 @@ defmodule Game do
   def get_marker(game), do: game.player_2_mark
 
   def update_board(board, game), do: %{game | board: board}
+
+  def next_open(game), do: List.first(get_available_moves(game))
+
+  def handle_next_move(game) do
+    move = next_open(game)
+    player_move_list = determine_move_list(game)
+    game = update_available_moves(move, game)
+    game = update_player_move_list(game, player_move_list, move)
+    place_marker(move, game)
+  end
+
+  def best_move(game) do
+    # best_score = -100
+    # new_move = nil
+    game = assign_maximizer(game)
+
+    # Iterate through every cell
+    score_move_list =
+      Enum.map([1, 2, 3, 4, 5, 6, 7, 8, 9], fn move ->
+        IO.puts("\r\n\r\nTrying == #{move} ")
```

Comment:
i think you are going to reach the limits of printf debugging pretty quickly here - have you tried using the debugger at any point during the apprenticeship?